src/chainparams.cpp

```diff
@@ -175,7 +175,6 @@ class CMainParams : public CChainParams {
             pnSeed6_main, pnSeed6_main + ARRAYLEN(pnSeed6_main));
 
         fDefaultConsistencyChecks = false;
-        fRequireStandardPolicy = true;
```

Comment:
Are we deleting these because we are never going to allow testnet to be different for defaults in the future?

Reply:
Yes. Why would we? Testnet and mainnet should be compatible.